platform-includes/migration/javascript-v8/integrations-package-removal/javascript.nextjs.mdx

```diff
@@ -0,0 +1,20 @@
+### Removal of `@sentry/integrations` package
```

Comment:
As far as I can see this only exists for Next.js in this PR, but this also applies to other packages 🤔

Reply:
Yes this is because this is part of the next.js specific docs! The rest of the sdks rely on the simpler migration docs, but we'll move them over to the better one's one by one.